docs/lang/articles/misc/ndarray_android.md

```diff
@@ -0,0 +1,449 @@
+---
+sidebar_position: 10
+---
+
+# Run a Taichi Program using Ndarray on Android
+
+Taichi's JIT (Just In Time) module compiles a Taichi kernel to the compute shaders according to the specified backend (`arch=`) and executes these shaders in Taichi's JIT runtime. Taichi's AOT (Ahead of Time) module, however, builds and saves the necessary compute shaders so that you can write your own runtime at a later point to load and execute these shaders on a platform without a Python environment.
+
+Taking a simulation of celestial bodies' orbits as an example, this tutorial walks you through the process of running a Taichi program using Ndarray on Android.
+
+> - Taichi kernels and compute shaders are *not* one-to-one mapping. Each Taichi kernel can generate multiple compute shaders, the number *usually* in line with the number of the for loops in the kernel.
```

Comment:
```suggestion
> - Taichi kernels and compute shaders are *not* one-to-one mapping. Each Taichi kernel can generate multiple compute shaders, the number is *usually* in line with that of the loops in the kernel.
```

Technically speaking, this is not in line. What about "on par with"?

Reply:
the number usually comparable to that of the loops in the kernel.